src/viztracer/patch.py

```diff
@@ -51,7 +51,7 @@ def build_command(args: Sequence[str]) -> list[str] | None:
                         py_args.append(f"-{cm_py_args}")
                     mode = [f"-{cm}"]
                     # -m mod | -mmod
-                    mode.append(cm_arg or next(args_iter, None))
+                    mode.append(cm_arg or next(args_iter))
```

Comment:
This is not a cosmetic change - this changes the behavior when `args_iter` exhausted. Is there a reason behind it? Is `mypy` complaining about it?

Reply:
Yes – the type of `mode` is implicitly `list[str]`, so you can't plonk a None in there.

```
src/viztracer/patch.py:54: error: Argument 1 to "append" of "list" has incompatible type "str | Any | None"; expected "str"  [arg-type]
```